nixos/modules/services/torrent/opentracker.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.opentracker;
+
+  out = {
+    options.services.opentracker = api;
+    config = lib.mkIf cfg.enable imp;
+  };
+
+  api = {
+    enable = mkEnableOption "opentracker";
+
+    package = mkOption {
```

Comment:
To use the latest version of opentracker without the need to override the package with nixpkgs.config.packageOverrides.

Right now the nixpkgs opentracker package is 2 years (2014-08-03 vs 2016-08-02) behind the latest commit and this option provides the ability to use a more current version instead.